component/org.wso2.carbon.identity.oauth2.grant.token.exchange/src/main/java/org/wso2/carbon/identity/oauth2/grant/token/exchange/Constants.java

```diff
@@ -61,4 +61,10 @@ public static class ConfigElements {
         public static final String IAT_VALIDITY_PERIOD_IN_MIN = "IATValidityPeriod";
         public static final String SUPPORTED_GRANT_TYPES = "SupportedGrantTypes";
     }
+
+    public enum SendLocalUser {
```

Comment:
better rename this Enum, may be to UserLinkStratergy, and rename the values also to reflect the behaviour

Reply:
addressed with [9faa855](https://github.com/wso2-extensions/identity-oauth2-grant-token-exchange/pull/20/commits/9faa855222dd5ba4a3349ced5d28388b6c33a8ff)